english/dansescape/dancEscape.md

```diff
@@ -0,0 +1 @@
+Dan's escape is quite simple. He puts together everything he learned in the course and quietly creates a link, file and other necessary components to a successful, if minor contribution to the 'story'. Dan's journey isn't supposed to end here. It's just beginning. He'll take what he learned about git and git hub and use them back in college.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.